src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
 # configure the mongoDB client for a single instance on localhost
-quarkus.mongodb.connection-string = mongodb://localhost:27017
\ No newline at end of file
+quarkus.mongodb.connection-string = mongodb://localhost:27017
+
+# mandatory if you don't specify the name of the database using @MongoEntity
+quarkus.mongodb.database = measure-shelter
```

Comment:
O seu mongodb conecta direitinho no localhost ne? Mesmo assim, não vamos padronizar os containeres no docker-compose?

Reply:
Esse arquivo não era para ter subido, dei um git add . na força do hábito kkkk, desculpa, mas iremos padronizar com o docker-compose sim.